automatic cert renewal for observability

dockerymick opened the pull request:
[Doc issue](https://github.com/open-cluster-management/backlog/issues/7125)

[Dev issue](https://github.com/open-cluster-management/backlog/issues/4766)

dockerymick committed:
automatic cert renewal for observablity

marcolan018 commented on security/certificates.adoc:
```diff
@@ -44,7 +44,7 @@ oc delete secret grc-0c925-grc-secrets -n open-cluster-management
 == Refresh certificates for {product-title}
 
-You can refresh all certificates that are issued by the {product-title-short} CA. During the refresh, the Kubernetes secret that is associated with each `cert-manager` certificate is deleted. The service restarts automatically to use the certifcate. Run the following command:
+You can refresh all certificates that are issued by the {product-title-short} CA. During the refresh, the Kubernetes secret that is associated with each `cert-manager` certificate is deleted. The service restarts automatically to use the certificate. Run the following command:
```
user don't need to run any command for cert renew, or restart any service to apply the renewed cert

dockerymick replied:
This file is for RHACM certificates. I was updating the spelling on line 47. @marcolan018 may you review the blurb that i added in the observability/observe_intro.adoc  file?

dockerymick committed:
Merge branch '2.2_stage' into 7125

dockerymick committed:
Update observe_intro.adoc

Renew to refresh 
https://github.com/open-cluster-management/backlog/issues/7125#issuecomment-756225642

dockerymick committed:
Update observe_intro.adoc

Changing refresh back to renew after receiving the description from Gus: https://github.com/open-cluster-management/backlog/issues/8366#issuecomment-758004172

dockerymick commented on observability/observe_intro.adoc:
```diff
@@ -20,7 +20,20 @@ When the service is enabled the `observability-endpoint-controller` is automatic
+
+* Manifest work for observability is updated.
+
+// is manifest work the multicluster-observability-operator yaml, the configured object store?
```
@marcolan018 can you clarify what the manifest work? Are there specific YAML files that we need to mention when we let the user know what work is being updated?

marcolan018 replied:
manifestwork is about the internal implementation about how the artifacts in hub side pushed to managed clusters. Can we update this step to something like "The renewed certificates are pushed to managed clusters by ACM"?

dockerymick committed:
Update observe_intro.adoc

updated after Marco's review

dockerymick commented:
@marcolan018 I have updated the file, may you rereview the PR?

marcolan018 reviewed (approved):
lgtm

dockerymick committed:
Update whats_new.adoc

updated refresh to renewed

dockerymick committed:
Update observe_intro.adoc

removed comment
